src/app/routes/User/index.js

```diff
@@ -0,0 +1,29 @@
+import React, { Fragment } from "react";
+import UserLayout from "./components/UserLayout";
+import * as ROUTES from "app/constants/routes";
+import withAuthorization from "../../session/withAuthorization";
+import Breadcumb from "../../components/Breadcumb";
+import Navigation from "../../components/Navigation";
+import Dashboard from "./containers/Dashboard";
+import Profile from "./containers/Profile";
+const UserDashboard = ({ authUser, history }) => {
+  const renderRightPane = () => {
+    switch (history.location.pathname.replace("/", "")) {
+      case ROUTES.USER_ROUTES.PROFILE.replace("/", ""):
+        return <Profile user={authUser} />;
+      default:
+        return <Dashboard user={authUser} />;
+    }
+  };
+
+  return (
+    <Fragment>
+      <Navigation />
+      <Breadcumb />
+      <UserLayout>{renderRightPane()}</UserLayout>
+    </Fragment>
+  );
+};
+const condition = authUser => !!authUser;
```

Comment:
!! có ý nghĩa gì

Reply:
dạ chỗ ni em pass điều kiện là authUser null hoặc not null thì trả về true false ạ.
Hàm nhận vào authUser , return về boolean>
nếu authUser !== null     =>    !authUser là false  => !!authUser ==> return về true.
nếu authUser == null => !authUser là true !!authUser ==> return về false .

Reply:
vì sao không dùng luôn authUser

Reply:
authUser là 1 object anh. em làm hàm condition vì để lấy boolean authUser có giá trị hay không ạ.